dart/add-created-at/main.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:dart_appwrite/dart_appwrite.dart';
+import 'dart:io';
+import 'dart:convert';
+
+void main() async {
+Map<String, String> environmentVariables = Platform.environment;
+final Client client = Client();
+client
+.setEndpoint(environmentVariables['APPWRITE_ENDPOINT'])
+.setProject(environmentVariables['APPWRITE_PROJECT_ID'])
```

Comment:
```suggestion
.setProject(environmentVariables['APPWRITE_FUNCTION_PROJECT_ID'])
```

This variable is automatically passed to every Appwrite Cloud function